File: Python/Day23.py
 def levelOrder(self,root):
        queue=[]
        if root:
  	         queue=[root]
                
        while len(queue) > 0:
            nd=queue.pop()
            if nd.left:
                queue.insert(0,nd.left)
            if nd.right:
                queue.insert(0,nd.right)
            print nd.data,
            

File: C++/Day23.cpp
void levelOrder(Node * root){
  queue<Node*> Q;
  Q.push(root);
  while(Q.size()!=0){
      Node* x=Q.front();
      cout<<x->data<<" ";
      Q.pop();
      if(x->left!=NULL){
          Q.push(x->left);
      } 
      if(x->right!=NULL){
          Q.push(x->right);
      }
  }
}
